doc/ref/cli/compile-forth.md

```diff
@@ -0,0 +1,22 @@
+## Name
+Disassemble Pravda programs.
+
+## Synopsys
+```
+pravda compile disasm [--input <filename>] [--output <filename>]
+```
+
+## Description
+
+Input file is a Pravda executable binary. Output is a text file with
+Pravda assembly code. By default read from stdin and print to stdout.
+            
+## Options
+
+|Option|Description|
```

Comment:
Mabye we can hide this header when there's no options? Or write something like `No options provided`.

Reply:
Done!